File: unittest/scripts/auto/py_devapi/scripts/mysqlx_collectionfind_help.py
shell.connect(__uripwd)
schema = session.create_schema('collectionfind_help')
coll = schema.create_collection('sample')
collfind = coll.find();
session.close()

#@ collfind
collfind.help()

#@ global ? for CollectionFind[USE:collfind]
\? CollectionFind

#@ global help for CollectionFind[USE:collfind]
\help CollectionFind

#@ collfind.bind
collfind.help('bind')

#@ global ? for bind[USE:collfind.bind]
\? CollectionFind.bind

#@ global help for bind[USE:collfind.bind]
\help CollectionFind.bind

#@ collfind.execute
collfind.help('execute')

#@ global ? for execute[USE:collfind.execute]
\? CollectionFind.execute

#@ global help for execute[USE:collfind.execute]
\help CollectionFind.execute

#@ collfind.fields
collfind.help('fields')

#@ global ? for fields[USE:collfind.fields]
\? CollectionFind.fields

#@ global help for fields[USE:collfind.fields]
\help CollectionFind.fields

#@ collfind.find
collfind.help('find')

#@ global ? for find[USE:collfind.find]
\? CollectionFind.find

#@ global help for find[USE:collfind.find]
\help CollectionFind.find

#@ collfind.group_by
collfind.help('group_by')

#@ global ? for group_by[USE:collfind.group_by]
\? CollectionFind.group_by

#@ global help for group_by[USE:collfind.group_by]
\help CollectionFind.group_by

#@ collfind.help
collfind.help('help')

#@ global ? for help[USE:collfind.help]
\? CollectionFind.help

#@ global help for help[USE:collfind.help]
\help CollectionFind.help

#@ collfind.limit
collfind.help('limit')

#@ global ? for limit[USE:collfind.limit]
\? CollectionFind.limit

#@ global help for limit[USE:collfind.limit]
\help CollectionFind.limit

#@ collfind.lock_exclusive
collfind.help('lock_exclusive')

#@ global ? for lock_exclusive[USE:collfind.lock_exclusive]
\? CollectionFind.lock_exclusive

#@ global help for lock_exclusive[USE:collfind.lock_exclusive]
\help CollectionFind.lock_exclusive

#@ collfind.lock_shared
collfind.help('lock_shared')

#@ global ? for lock_shared[USE:collfind.lock_shared]
\? CollectionFind.lock_shared

#@ global help for lock_shared[USE:collfind.lock_shared]
\help CollectionFind.lock_shared

#@ collfind.sort
collfind.help('sort')

#@ global ? for sort[USE:collfind.sort]
\? CollectionFind.sort

#@ global help for sort[USE:collfind.sort]
\help CollectionFind.sort

shell.connect(__uripwd)
session.drop_schema('collectionfind_help');
session.close()

File: unittest/scripts/auto/js_devapi/scripts/mysqlx_collectionfind_help.js
//@ Initialization
shell.connect(__uripwd);
var schema = session.createSchema('collectionfind_help');
var collection = schema.createCollection('sample');
var crud = collection.find()
session.close();

//@ CollectionFind help
crud.help();

//@ CollectionFind help, \? [USE:CollectionFind help]
\? CollectionFind

//@ Help on bind
crud.help('bind');

//@ Help on bind, \? [USE:Help on bind]
\? CollectionFind.bind

//@ Help on execute
crud.help('execute');

//@ Help on execute, \? [USE:Help on execute]
\? CollectionFind.execute

//@ Help on fields
crud.help('fields');

//@ Help on fields, \? [USE:Help on fields]
\? CollectionFind.fields

//@ Help on find
crud.help('find');

//@ Help on find, \? [USE:Help on find]
\? CollectionFind.find

//@ Help on groupBy
crud.help('groupBy');

//@ Help on groupBy, \? [USE:Help on groupBy]
\? CollectionFind.groupBy

//@ Help on having
crud.help('having');

//@ Help on having, \? [USE:Help on having]
\? CollectionFind.having

//@ Help on help
crud.help('help');

//@ Help on help, \? [USE:Help on help]
\? CollectionFind.help

//@ Help on limit
crud.help('limit');

//@ Help on limit, \? [USE:Help on limit]
\? CollectionFind.limit

//@ Help on lockExclusive
crud.help('lockExclusive');

//@ Help on lockExclusive, \? [USE:Help on lockExclusive]
\? CollectionFind.lockExclusive

//@ Help on lockShared
crud.help('lockShared');

//@ Help on lockShared, \? [USE:Help on lockShared]
\? CollectionFind.lockShared

//@ Help on offset
crud.help('offset');

//@ Help on offset, \? [USE:Help on offset]
\? CollectionFind.offset

//@ Help on skip
crud.help('skip');

//@ Help on skip, \? [USE:Help on skip]
\? CollectionFind.skip

//@ Help on sort
crud.help('sort');

//@ Help on sort, \? [USE:Help on sort]
\? CollectionFind.sort

//@ Finalization
shell.connect(__uripwd);
session.dropSchema('collectionfind_help');
session.close();
